lib/realtime/supervisor.ex

```diff
@@ -5,11 +5,15 @@ defmodule Realtime.Supervisor do
     Supervisor.start_link(__MODULE__, :ok)
   end
 
+  @spec registry_name() :: Registry.registry()
+  def registry_name(), do: Realtime.Registry
+
   @impl true
   def init(:ok) do
     children = [
-      {Registry, keys: :duplicate, name: Realtime.Server.registry_name()},
+      {Registry, keys: :duplicate, name: registry_name()},
```

Comment:
Do you need two separate registries? Maybe a pattern like [events.ex](https://github.com/mbta/api/blob/master/apps/events/lib/events.ex) where clients provide the key they're registering for would work, with keys like `:data_status` and `{:route, ID}`.

Reply:
It is just one registry. The key is the pid of the pubsub genserver (`server.ex` or `data_status_registry.ex`).

Reply:
Ah, can we give it a different name then? Maybe `DataStatusServer`?

Reply:
`DataStatusPubsub`?